daemon/cluster/convert/container.go

```diff
@@ -302,6 +302,7 @@ func containerToGRPC(c *types.ContainerSpec) (*swarmapi.ContainerSpec, error) {
 		CapabilityAdd:  c.CapabilityAdd,
 		CapabilityDrop: c.CapabilityDrop,
 		Ulimits:        ulimitsToGRPC(c.Ulimits),
+		OomScoreAdj:    c.OomScoreAdj,
```

Comment:
This covers `containerToGRPC`, but you need a matching back-conversion in `containerSpecFromGRPC`. Without the back-conversion, the client cannot correctly read out the ServiceSpec.

Reply:
Thanks, this was an IDE issue.  Goland just removed code without notice, I have to turn off a few options.